packages/arb-validator/validator/validator.go

```diff
@@ -364,12 +382,25 @@ func (validator *Validator) Run(ctx context.Context, recvChan <-chan ethbridge.N
 			case ethbridge.VMEvent:
 				err := validator.eventUpdate(ev, notification.Header)
 				if err != nil {
-					log.Println("Error processing event update", err)
+					//log.Printf("*****Validator %v: error - %v\n", validator.Name, err)
+					if errstat, ok := err.(*bridge.Error); ok {
```

Comment:
If this is the only type of error that can be returned from `eventUpdate` it'd probably be better to change the signature and avoid this cast